src/plugins/files/server/routes/common.ts

```diff
@@ -19,11 +19,7 @@ export function getDownloadHeadersForFile({ file, fileName }: Args): ResponseHea
   return {
     'content-type':
       (fileName && mime.getType(fileName)) ?? file.data.mimeType ?? 'application/octet-stream',
-    // Note, this name can be overridden by the client if set via a "download" attribute on the HTML tag.
-    'content-disposition': `attachment; filename="${fileName || getDownloadedFileName(file)}"`,
     'cache-control': 'max-age=31536000, immutable',
-    // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options
-    'x-content-type-options': 'nosniff',
```

Comment:
The header fields `content-disposition` and `x-content-type-options` are removed because the handler res.file already takes care of providing these values.